media/css/pydici.css

```diff
@@ -12,13 +12,32 @@ body {
     padding: 0px;
 }
 
-.table-pydici-extra-condensed thead td {
+
+.table-pydici-extra-condensed thead {
```

Comment:
why did you removed td ? This css class is used by other tables

Reply:
mistake by my part! I readded the td